.goreleaser.yml

```diff
@@ -43,3 +43,14 @@ brew:
   test: |
     system "#{bin}/trellis --autocomplete-install"
     system "#{bin}/trellis -v"
+snapcraft:
+  name: trellis-cli
```

Comment:
🤔 we'll need to figure out what the binary will be called since we want it to be `trellis` but the package name should be `trellis-cli`.